### PR TITLE
Update

### PR DESCRIPTION
**Pull request by marcusellison**
- [x] added search updates
- [x] added interface for newUpdates
- [x] refactored actionbar create update call
- [x] add texupd api create call
- [x] move id keys into own module
- [x] Fix update validation character limit
- [x] get successful create update response
- [x] get successful mutate update response
- [x] fix getting response ids by index in array
- [x] remove lint errors

### CODE REVIEW

File: src/module/api/texupd/create/index.ts [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  CreateI_Obj,
+  CreateI_Obj_Property,
+} from 'module/api/texupd/proto/create_pb';
+import { APIClient } from 'module/api/texupd/proto/ApiServiceClientPb';
+import {INewUpdate} from 'module/interface/update'
+
+const AUDIENCEIDKEY = 'audience.venturemark.co/id';
+const ORGANIZATIONIDKEY = 'organization.venturemark.co/id';
+const TIMELINEIDKEY = "timeline.venturemark.co/id";
+const USERIDKEY = 'user.venturemark.co/id';
```
Comment (xh3b4sd) [inline src/module/api/texupd/create/index.ts:13]: It would be cool if we could put these magic strings into one common place so that we can use shared code and not duplicate it all over the place.

File: src/module/api/texupd/proto/ApiServiceClientPb.ts [hunk 2]
```diff
@@ -11,10 +11,10 @@
...
 import * as grpcWeb from 'grpc-web';
 
-import * as pbf_texupd_create_pb from '../../pbf/texupd/create_pb';
-import * as pbf_texupd_delete_pb from '../../pbf/texupd/delete_pb';
-import * as pbf_texupd_search_pb from '../../pbf/texupd/search_pb';
-import * as pbf_texupd_update_pb from '../../pbf/texupd/update_pb';
+import * as pbf_texupd_create_pb from './create_pb';
+import * as pbf_texupd_delete_pb from './delete_pb';
+import * as pbf_texupd_search_pb from './search_pb';
+import * as pbf_texupd_update_pb from './update_pb';
```
Comment (xh3b4sd) [inline src/module/api/texupd/proto/ApiServiceClientPb.ts:17]: This is the generated code and should not be touched. We need to collect the reasons for these modifications so that when we get to fix the code generation we can incorporate the necessary changes in our automation.

File: src/module/interface/update/index.ts [hunk 3]
```diff
@@ -1,11 +1,16 @@
...
-  isContext: boolean;
 }
+
+export interface INewUpdate {
+  text: string;
+  audienceId: string,
+  organizationId: string,
+  timelineId: string;
+  userId: string;
+}
```
Comment (xh3b4sd) [inline src/module/interface/update/index.ts:16]: Please make sure to make your editor keep the newlines at the end of files. That is like an unwritten law. The next developer will revert the change automatically. We can spare the back and forth. ❤️

File: src/module/api/texupd/proto/ApiServiceClientPb.ts (hunk 2 above)
Comment (marcusellison) [reply src/module/api/texupd/proto/ApiServiceClientPb.ts:17]: ok. In this case these files are in the same folder.

File: src/module/interface/update/index.ts (hunk 3 above)
Comment (marcusellison) [reply src/module/interface/update/index.ts:16]: I just started using the VS Code editor - will make adjustment.

File: src/module/api/texupd/proto/ApiServiceClientPb.ts (hunk 2 above)
Comment (xh3b4sd) [reply src/module/api/texupd/proto/ApiServiceClientPb.ts:17]: So did you fix or break it? Was the generated code broken or working? I am not following this change and the history of events.

Comment (marcusellison) [reply src/module/api/texupd/proto/ApiServiceClientPb.ts:17]: The change fixes the import statement because the src/module/api/texupd/proto/ApiServiceClientPb.ts file is importing from files that are in the same directory.

File: src/component/actionbar/index.tsx [hunk 4]
```diff
@@ -17,12 +26,77 @@ interface ActionBarProps extends DefaultActionBarProps {
...
+    if (!editorShape.hasContent) {
+      const error = 'Please enter some text';
+      setEditorShape({ ...editorShape, error });
+      return;
+    }
+
+    if (serialize(editorShape.value).length > 241) {
+      const error = `Your update is ${
+        serialize(editorShape.value).length
+      } characters. The limit is 240 characters`;
```
Comment (xh3b4sd) [inline src/component/actionbar/index.tsx:68]: The backend defines 280 right now.
